app/views/admin/pages/sections/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <% @page_title = 'Edit section' %>
 
-<%= form_for( @section, url: "/admin/page/section/#{@section.id}", method: 'post' ) do |f| %>
+<%= form_for @section, url: "/admin/pages/section/#{@section.id}", method: 'post' do |f| %>
```

Comment:
Prefer to use route helper methods instead of hard-coding the paths, e.g.

```diff
-<%= form_for @section, url: "/admin/pages/section/#{@section.id}", method: 'post' do |f| %>
+<%= form_for @section, url: admin_pages_section_path(@section), method: 'post' do |f| %>
```

`config/routes.rb` might need tweaking to support the above method. Run `rails routes` to determine the correct `*_path` method name.

Reply:
It doesn't seem to have one. Probably because [ see below ].

Reply:
How about this style?

`<%= form_for( @page, url: url_for( controller: '/admin/pages', action: 'edit', id: @page.id ), method: 'post' ) do |f| %>`

Reply:
> It doesn't seem to have a suggested path for any of the edit methods

Ah right, add an `as: :some_name` option to the route to generate a named helper method, e.g.

```rb
get  'section/:id', to: 'sections#edit', as: :section
```

This will generate an `admin_pages_section_path` helper.

Reply:
Oooh. Shiny. :)